### PR TITLE
6 show reviews

### LINKED ISSUE #6: Show reviews on Restaurant list page
As a User
So I can decide which restaurant to visit
I want to see all reviews on the list of restaurants

### PR DESCRIPTION
**Pull request by AttyC**
Closes #6

### CODE REVIEW

File: app/assets/stylesheets/restaurants.scss [hunk 1]
```diff
@@ -1,3 +1,11 @@
 // Place all the styles related to the Restaurants controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+table {
```
Comment (rcvink) [inline app/assets/stylesheets/restaurants.scss:4]: :+1:

File: app/views/restaurants/index.html.erb [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
+   <td><%= restaurant.description %></td>
+
+   <td id="reviews"><% restaurant.reviews.each do |review| %>
+     (<%= review.rating %>) <%= review.comment  %>
+   <% end %>
+ </td>
+
+ </tr>
+ <% end %>
 </table>
```
Comment (rcvink) [inline app/views/restaurants/index.html.erb:23]: :v:

File: spec/features/display_reviews_spec.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe "Display Reviews", :type => :feature do
+  feature 'Show reviews for one restaurant ' do
+    scenario 'when on restaurant index page' do
+      create_restaurant('Testaurant', 'Delicious')
+      create_review(2, 'really good!!')
```
Comment (rcvink) [inline spec/features/display_reviews_spec.rb:7]: really good!!

**Review by rcvink — APPROVED** [review]
:cat2:

**Review by ealitten — APPROVED** [review]

**Pull request merged**